### ISSUE TITLE
Allow burst speeds

### ISSUE THREAD

**Issue by jtagcat** [magnific0/wondershaper#58]
I would like to throttle long-term usage of bandwidth.

**Comment by magnific0** [comment]
This sounds like a great feature, but not something wondershaper is capable of. Wondershaper targets the entire interface, not single connections or processes. I'm quite sure there are different tools which have that as a starting point that should be more suitable as a starting point.

**Comment by jtagcat** [comment]
iirc, wondershaper is on top of tc, and tc has bursts?

**Comment by magnific0** [comment]
You are right, I misunderstood your question. Reopening and tagging as enhancement. I'll probably won't work on it, but other's can chime in on this feature and I'm open to pull requests.
